gapis/api/transform2/command_id.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package transform2 contains the elements to be able to transform
+// commands which consist of interfaces for individual transform operations
+// and a transform chain to run all of them.
+package transform2
+
+import (
+	"github.com/google/gapid/gapis/api"
+)
+
+type CommandType uint32
+
+const (
```

Comment:
BeforeCommands, RealCommand, AfterCommands? The names here are a little confusing.

Reply:
Open to any suggestions.

Reply:
I though about this but if we change this, now we get a map that beginTransform() produces BeforeCommand, commandGenerator produces RealCommand and EndTransform() produces AfterCommand which I do not think it makes it better and actually makes it even more confusing. 

If you disagree, we can talk about renaming when you are back and we can change it with a later pr. It is a small change to implement anyway.